src/app/shared/services/studio-configuration.ts

```diff
@@ -0,0 +1,14 @@
+interface BlueprintConfiguration {
+  ServerPostrollDuration: number
+}
+
+interface Settings {
+  mediaPreviewUrl: string
+}
+
+export interface StudioConfiguration {
+  data: {
+    settings: Settings
+    blueprintConfiguration: BlueprintConfiguration
+  }
+}
```

Comment:
The `data` is not part of the `StudioConfiguration`. It's part of the `Response` object that we receive from the backend.
This interface should just be:
```
export interface StudioConfiguration {
  settings: Settings
  blueprintConfiguration: BlueprintConfiguration
}
```